app/screens/likerland-oauth-screen/likerland-oauth-screen.tsx

```diff
@@ -126,22 +129,30 @@ export class LikerLandOAuthScreen extends React.Component<
     this.handleURLChange(url)
   }
 
-  private onWebviewError = () => {
+  private showErrorAlert = (message?: string) => {
+    if (this.hasShownErrorAlert) return
+    this.hasShownErrorAlert = true
```

Comment:
Would `hasShownErrorAlert` reset to false if I retry?

Reply:
No retry for the error alert, the user will be returned to the initial screen to try again, and enter this screen again with the initial state